pkg/color/formatter.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2019 The Skaffold Authors
```

Comment:
this line should look something like `Copyright 2019 Google Inc. All rights reserved.`